web/satellite/tests/unit/store/usage.spec.ts

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+import Vuex from 'vuex';
+import { createLocalVue } from '@vue/test-utils';
+import { ProjectUsageApiGql } from '@/api/usage';
+import { makeUsageModule, PROJECT_USAGE_ACTIONS, PROJECT_USAGE_MUTATIONS } from '@/store/modules/usage';
+import { DateRange, ProjectUsage } from '@/types/usage';
+import { ProjectsApiGql } from '@/api/projects';
+import { makeProjectsModule } from '@/store/modules/projects';
+import { Project } from '@/types/projects';
+
+const Vue = createLocalVue();
+const projectUsageApi = new ProjectUsageApiGql();
+const usageModule = makeUsageModule(projectUsageApi);
+
+const projectsApi = new ProjectsApiGql();
+const projectsModule = makeProjectsModule(projectsApi);
+const selectedProject = new Project('', '', '', '');
+selectedProject.id = '1';
+projectsModule.state.selectedProject = selectedProject;
+
+let testDate1 = new Date();
+testDate1.setDate(1);
+let testDate2 = new Date();
+testDate2.setDate(2);
+const testUsage = new ProjectUsage(2, 3, 4, testDate1, testDate2);
+const now = new Date();
+
+Vue.use(Vuex);
+
+const store = new Vuex.Store({ modules: { usageModule, projectsModule } });
+
+const state = (store.state as any).usageModule;
+
+describe('mutations', () => {
+    beforeEach(() => {
+        createLocalVue().use(Vuex);
+    });
+
+    it('fetch project usage', () => {
+        store.commit(PROJECT_USAGE_MUTATIONS.FETCH, testUsage);
+
+        expect(state.projectUsage.storage).toBe(2);
+        expect(state.projectUsage.egress).toBe(3);
+        expect(state.projectUsage.objectCount).toBe(4);
+        expect(state.startDate.toDateString()).toBe(now.toDateString());
```

Comment:
If you use different place with ``new Date()`` initialization, its value may differs